### PR TITLE
packaging: fix duplicate share directories (use only lowercase)

### PR DESCRIPTION
**Pull request by Ryex**
`share/PrismLauncher/` & `share/prismlauncher/`  -> only `share/PrismLauncher`

### CODE REVIEW

**Review by getchoo — APPROVED** [review]

**Comment by Scrumplex** [comment]
I think I might prefer the opposite, actually. Storing our JARs in PrismLauncher instead of prismlauncher

**Comment by Ryex** [comment]
> I think I might prefer the opposite, actually. Storing our JARs in PrismLauncher instead of prismlauncher

that's a larger change and may break some automated packaging. but it was also my first move that I revised to this....

**Comment by Scrumplex** [comment]
As it is right now your change makes lines 427 to 432 in `Application.cpp` useless. I just thought that that's more elegant than our manual way of figuring out the path

**Comment by getchoo** [comment]
> may break some automated packaging

what packaging exactly? most distributions either a) don't care about paths are outputted or b) make you specify them anyways. it's also not like the removal of `share/PrismLauncher` wouldn't have a similar affect frankly

**Comment by Ryex** [comment]
Moved it to use PascalCase folder name

File: launcher/Application.cpp [hunk 1]
```diff
@@ -1567,7 +1571,7 @@ QString Application::getJarPath(QString jarFile)
 {
     QStringList potentialPaths = {
 #if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD) || defined(Q_OS_OPENBSD)
-        FS::PathCombine(m_rootPath, "share/" + BuildConfig.LAUNCHER_APP_BINARY_NAME),
+        FS::PathCombine(m_rootPath, "share/" + BuildConfig.LAUNCHER_NAME),
```
Comment (Scrumplex) [inline launcher/Application.cpp]:
```suggestion
        FS::PathCombine(m_rootPath, "share", BuildConfig.LAUNCHER_NAME),
```

**Review by Scrumplex — APPROVED** [review]
Long term we probably want to use `KDE_INSTALL_DATADIR` and use QStandardPaths to locate the jar files.

**Pull request merged**